news/648.bugfix

```diff
@@ -0,0 +1 @@
+OmegaConf.create() was ignoring the `parent` argument when taking a string as input
```

Comment:
```suggestion
Fix OmegaConf.create() to set the provided `parent` when creating a config from a dict, list and a YAML string.
```

Reply:
Actually the bug is only for YAML strings => check new text in 59b7bdd